src/pages/admin/teacher-complete/components/Header.tsx

```
@@ -0,0 +1,182 @@
+                    transform: isOpen ? "rotate(180deg)" : "rotate(0deg)",
+                    transition: "transform 0.2s",
+                  }}>
+                  <DownArrowDropdownIc />
+                </span>
+              </CustomSelectButton>
+              {isOpen && (
+                <CustomDropDown>
+                  {yearOptions.map((option) => (
+                    <CustomOption {...getOptionProps(option)}>{option.label}</CustomOption>
+                  ))}
+                </CustomDropDown>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)